fuelup-init.sh

```diff
@@ -226,8 +226,8 @@ downloader() {
         fi
         if [ -n "$_err" ]; then
             echo "$_err" >&2
-            if echo "$_err" | grep -q 404$; then
-                err "installer for platform '$3' not found, this may be unsupported"
+            if echo "$_err" | grep -q 404; then
+		err "fuelup was not found - either the release is not ready yet or the tag is invalid. You can check if the release is available here: https://github.com/FuelLabs/fuelup/releases"
```

Comment:
Indentation?

Reply:
are spaces preferred here? I ran it all under a shell formatter one of the last PRs. Can we disregard this indentation issue this PR and I will open another PR running `shfmt`? Would probably be worth including in #31 as well